pkgs/by-name/pa/package-assistant/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "package-assistant";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "patryk-s";
+    repo = pname;
+    rev = version;
```

Comment:
`"refs/tags/${version}` is better, as that is safe if there were to be a similarly named branch

Reply:
sure, makes sense, thanks for the tip.